### PR TITLE
Update docs (Minio, Rook, NFS)

### PR DESCRIPTION
**Pull request by hossainemruz**
Tasks:
- [x] Move Minio doc under platforms
- [x] Move Rook doc under platforms
- [x] Use NFS volume instead of `hostPath`

### CODE REVIEW

File: docs/guides/backup.md [hunk 1]
```diff
@@ -43,23 +43,23 @@ namespace/demo created
...
 5. Finally, `stash` sidecar container takes periodic backup of the volume to specified backend. It also creates a `Repository` crd in first backup which represents the original repository in the backend in Kubernetes native way.
 
 ## Backup
 
-In order to take backup, we need some sample data. Stash has some sample data in [stash-data](https://github.com/appscode/stash-data) repository. As [gitRepo](https://kubernetes.io/docs/concepts/storage/volumes/#gitrepo) volume has been deprecated, we will not use this repository as volume directly. Instead, we will create a [configMap](https://kubernetes.io/docs/concepts/storage/volumes/#configmap) from these data and use that ConfigMap as data source.
+In order to take back up, we need some sample data. Stash has some sample data in [stash-data](https://github.com/appscode/stash-data) repository. As [gitRepo](https://kubernetes.io/docs/concepts/storage/volumes/#gitrepo) volume has been deprecated, we are going to not use this repository as volume directly. Instead, we are going to create a [configMap](https://kubernetes.io/docs/concepts/storage/volumes/#configmap) from these data and use that ConfigMap as the data source.
```
Comment (hossainemruz) [inline docs/guides/backup.md]: `we will not use` sounds better than `we are going to not use`. you can use `we are not going to use`.

File: docs/guides/platforms/gke.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ menu_name: product_stash_0.7.0
 
 # Using Stash with Google Kubernetes Engine (GKE)
 
-This tutorial will show you how to use Stash to **backup** and **restore** a Kubernetes deployment in [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/). Here, we are going to backup the `/source/data` folder of a busybox pod into  [GCS bucket](/docs/guides/backends.md#google-cloud-storage-gcs). Then, we will show how to recover this data into a `gcePersistentDisk` and `PersistentVolumeClaim`. We will also re-deploy deployment using this recovered volume.
+This tutorial will show you how to use Stash to **backup** and **restore** a Kubernetes deployment in [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/). Here, we are going to back up the `/source/data` folder of a busybox pod into  [GCS bucket](/docs/guides/backends.md#google-cloud-storage-gcs). Then, we will show how to recover this data into a `gcePersistentDisk` and `PersistentVolumeClaim`. We will also re-deploy deployment using this recovered volume.
```
Comment (hossainemruz) [inline docs/guides/platforms/gke.md]: Won't  be `backup`?
[backup](https://en.oxforddictionaries.com/definition/backup)
[back up](https://en.oxforddictionaries.com/definition/back_up)

File: docs/guides/backup.md [hunk 3]
```diff
@@ -43,23 +43,23 @@ namespace/demo created
...
 5. Finally, `stash` sidecar container takes periodic backup of the volume to specified backend. It also creates a `Repository` crd in first backup which represents the original repository in the backend in Kubernetes native way.
 
 ## Backup
 
-In order to take backup, we need some sample data. Stash has some sample data in [stash-data](https://github.com/appscode/stash-data) repository. As [gitRepo](https://kubernetes.io/docs/concepts/storage/volumes/#gitrepo) volume has been deprecated, we will not use this repository as volume directly. Instead, we will create a [configMap](https://kubernetes.io/docs/concepts/storage/volumes/#configmap) from these data and use that ConfigMap as data source.
+In order to take back up, we need some sample data. Stash has some sample data in [stash-data](https://github.com/appscode/stash-data) repository. As [gitRepo](https://kubernetes.io/docs/concepts/storage/volumes/#gitrepo) volume has been deprecated, we are going to not usea  this repositoryolum e directly. Instead, we are going to create a a [configMap](ht/kub ernetes.io/docs/concepts/storage/volumes/#configmap) from these data and use that ConfigMap as the data source.
```
Comment (hossainemruz) [inline docs/guides/backup.md]: ` usea  this repositoryolum e ` --> `use this repository as volume` ?

File: docs/guides/backup.md [hunk 4]
```diff
@@ -433,9 +433,9 @@ NAME                    BACKUPCOUNT   LASTSUCCESSFULBACKUP   AGE
 deployment.stash-demo   4             23s                    4m
 ```
 
-Here, `BACKUPCOUNT` field indicates number of backup snapshot has taken in this repository.
+Here, `BACKUPCOUNT` field indicates the number of backup snapshots has taken in this repository.
 
-`Restic` will take backup of the volume periodically with a 1-minute interval. You can verify that backup snapshots has been created successfully by,
+`Restic` will take backup of the volume periodically with a 1-minute interval. You can verify that backup snapshots have been created successfully by, running the following command:
```
Comment (hossainemruz) [inline docs/guides/backup.md]: `successfully by, running the following command:`
comma after `by`

File: docs/guides/platforms/gke.md (hunk 2 above)
Comment (tamalsaha) [reply docs/guides/platforms/gke.md]: if verb, then `back up`.
if noun, `backup` .

**Pull request merged**